requirements.txt

```diff
@@ -1,5 +1,6 @@
 # core dependencies
 pytorch==1.3.0
+pillow==6.2.0
```

Comment:
I'm pretty sure you have to add it here too: https://github.com/kornia/kornia/blob/master/setup.py#L60